api.go

```diff
@@ -0,0 +1,17 @@
+package apidApigeeSync
+
+import (
+	"github.com/30x/apid-core"
+	"net/http"
+)
+
+const tokenEndpoint = "/accesstoken"
+
+func InitAPI(services apid.Services) {
+	services.API().HandleFunc(tokenEndpoint, getAccessToken).Methods("GET")
+}
+
+func getAccessToken(w http.ResponseWriter, r *http.Request) {
+	<-tokenUpdatedApiNotificationChan
+	w.Write([]byte(apidTokenManager.getBearerToken()))
+}
```

Comment:
this was quick and dirty, I will fix this up in the next commit to have proper block parameter, If-None-Match, etc